python/redmonster/physics/zpicker2.py

```diff
@@ -107,7 +108,7 @@ def classify_obj(self, zfindobjs, zfitobjs, flags, rchi2threshold=0.005):
             npixsteptuple = ()
             fstuple = ()
             # Catch spectra that are all 0's and return null result
-            if len(n.where(self.flux[ifiber] != 0)[0]) == 0 or len(n.where(self.ivar[ifiber] != 0)[0]) == 0:
+            if len(n.where(self.flux[0] != 0)[0]) == 0:
```

Comment:
I'm not sure why that was changed.  I put it back, though using a different method to check for all zeros: `if n.all(self.flux[ifiber] == 0.0) or n.all(self.ivar[ifiber] == 0.0):`